src/main/java/com/hcl/appscan/jenkins/plugin/scanners/StaticAnalyzer.java

```diff
@@ -222,6 +226,9 @@ public Map<String,String> getProperties(VariableResolver<String> resolver) {
             if(m_scanSpeed!=null && !m_scanSpeed.isEmpty() && getHasOptions()) {
```

Comment:
Cant we use isNullOrEmpty() here as well?

Reply:
Sure, Kripa.